Day_2/04_DifferentialExpression.Rmd

```diff
@@ -438,7 +479,7 @@ dgeList_Filtered %>%
 
 Next we might choose to perform a Principal Component Analysis on our data, commonly abbreviated to PCA.
 This time, let's take our CPM values & asses them on the log2 scale to make sure our PCA results are not heavily skewed by highly expressed genes.
-Our first step is to create a new object to hold the PCA results. In the following code chunk we take our filtered DGEList obeject `dgeList_Filtered` and pipe it to the `cpm()` function we used earlier to calculate the CPM values for each gene prior to setting the filtering threshold. We have made one small change here by including $log - TRUE$ in the function call. By default this is $log_2$. Next we pipe the $log2$CPM values into the `t()` function. This will _transpose_ the matrix of counts such that the matrix now has genes in the columns and samples in the rows, which we then pipe into `prcomp` from the `stats` package. `prcomp()` returns a list with class "prcomp" containing a number of useful components.
+Our first step is to create a new object to hold the PCA results. In the following code chunk we take our filtered DGEList obeject `dgeList_Filtered` and pipe it to the `cpm()` function we used earlier to calculate the CPM values for each gene prior to setting the filtering threshold. We have made one small change here by including $log = TRUE$ in the function call. By default this is $log_2$. Next we pipe the $log_2$CPM values into the `t()` function. This will _transpose_ the matrix of counts such that the matrix now has genes in the columns and samples in the rows, which we then pipe into `prcomp` from the `stats` package. `prcomp()` returns a list with class "prcomp" containing a number of useful components.
```

Comment:
- Breaks between sentences.
- Use code ticks `` ` `` rather than LaTeX math mode for parameters (``s/$log = TRUE$/`log = TRUE`/``, etc)